itest/litd_custom_channels_test.go

```diff
@@ -94,7 +94,30 @@ var (
 			"not_use_on_mainnet",
 		"--taproot-assets.experimental.rfq.mockoracleassetsperbtc=" +
 			"5820600",
+		"--taproot-assets.experimental.rfq.acceptpricedeviationppm=50000",
 	}...)
+
+	litdArgsTemplateDiffOracle = []string{
+		"--taproot-assets.allow-public-uni-proof-courier",
+		"--taproot-assets.universe.public-access=rw",
+		"--taproot-assets.universe.sync-all-assets",
+		"--taproot-assets.universerpccourier.skipinitdelay",
+		"--taproot-assets.universerpccourier.backoffresetwait=1s",
+		"--taproot-assets.universerpccourier.numtries=5",
+		"--taproot-assets.universerpccourier.initialbackoff=300ms",
+		"--taproot-assets.universerpccourier.maxbackoff=600ms",
+		"--taproot-assets.universerpccourier.skipinitdelay",
+		"--taproot-assets.universerpccourier.backoffresetwait=100ms",
+		"--taproot-assets.universerpccourier.initialbackoff=300ms",
+		"--taproot-assets.universerpccourier.maxbackoff=600ms",
+		"--taproot-assets.custodianproofretrievaldelay=500ms",
```

Comment:
nit: can use `append(litdArgsTemplateNoOracle, []string{` instead of duplicating all these values (since they look identical at first glance).